docs/unifiedpush/aerogear-push-android/google-setup.asciidoc

```diff
@@ -1,45 +1,40 @@
 
-=== Registering with Google
+* the Server key
+* the Sender ID
+* the `google-services.json` file that needs to be placed into the /app folder of your Android Studio project.
 
-First, we must authenticate with Google. Google offers the Google Cloud Messaging (GCM) platform, formerly known as Cloud to Device Messaging (C2DM). You will need a Google Account for this. Now open the https://cloud.google.com/console[Google Cloud Console].
+=== Project Setup and App Registration using the Firebase Console
 
+First, you must register with the Firebase Cloud Messaging (FCM) platform, formerly known as Google Cloud Messaging (GCM). You need a Google Account to do this. Start by opening the https://console.firebase.google.com[Firebase Console].
 
-- If you haven't created a project, click _Create Project_:
+- If you haven't created a project, click _Create New Project_:
 
-image:./img/gcc_1.png[Create New Project]
+image:./img/create_new_project.png[Create New Project]
 
-- Now give the new project a name:
+- Now give your new project a name:
 
-image:./img/gcc_2.png[New Project Name]
+image:./img/project_name.png[New Project Name]
 
-- Once the project has been created you will see the overview screen. From there grab the project number (used later as the _Sender ID_):
+- Once the project has been created you will be taken to the Overview screen. From there click the Settings icon located next to the project name in the sidebar, and in the pop-up menu click _Project Settings_.
 
-image:./img/gcc_3.png[Project Number]
+image:./img/project_settings.png[Project Number]
 
-Now you need to enable the _Google Cloud Messaging for Android_ APIs. To do that, navigate to +APIs & auth+ > +APIs+ and scroll down to the _Google Cloud Messaging for Android_ item: which you need to turn on:
+On the _Project Settings screen_, switch to the _Cloud Messaging_ tab, where you can find the _Server key_ and _Sender ID_ (known in GCM as _Project Number_).   
 
-image:./img/gcc_4_a.png[Cloud Messaging for Android]
+image:./img/retrieve_credentials.png[Retrieve Credentials]
 
-Now click in the link and press the _Enable API_ button:
+- As the final step of the setup procedure, you need to download the `google-services.json` file. To do this, first navigate to the Overview screen and click on _Add App_ (If you have no apps using FCM yet, select the option to _Add Firebase to your Android app_).
 
-image:./img/gcc_4_b.png[Enabling Cloud Messaging for Android]
+image:./img/add_app.png[Add Firebase to your app]
 
+- In the pop-up window enter the Package name of your app and then click on _Add app_. This automatically generates the `google-services.json` file that must be added to your app manifest in Android Studio. Download the .json file and save it. To close the pop-up window, click _Continue_ and then _Finish_ on the following screen.
```

Comment:
@rsmeral So, given that the /app folder may not be present in all development environments, would it be reasonable to just say here that the `google-services.json` file needs to be built into the client app binary?